src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -147,20 +145,20 @@ public KafkaTopicRepository(final KafkaZookeeper kafkaZookeeper,
         } catch (final InterruptException e) {
             Thread.currentThread().interrupt();
             circuitBreaker.markSuccessfully();
-            handlePublishingException(publishScope, item, "internal error");
+            handlePublishingException(publishSpan, item, "internal error");
             throw new EventPublishingException("Error publishing message to kafka", e);
         } catch (final RuntimeException e) {
             kafkaFactory.terminateProducer(producer);
             circuitBreaker.markSuccessfully();
-            handlePublishingException(publishScope, item, "internal error");
+            handlePublishingException(publishSpan, item, "internal error");
             throw new EventPublishingException("Error publishing message to kafka", e);
         }
     }
 
-    private void handlePublishingException(final Scope scope, final BatchItem batchItem, final String error) {
+    private void handlePublishingException(final Span span, final BatchItem batchItem, final String error) {
```

Comment:
please, remove this method, it is not actionable from user point of view, but it could waist resources